tests/jetty-testers/src/main/java/org/eclipse/jetty/tests/testers/JettyHomeTester.java

```diff
@@ -261,7 +262,7 @@ public static void unzip(Path archive, Path outputDir) throws IOException
                 }
             }
         }
-        catch (FileAlreadyExistsException e)
+        catch (FileAlreadyExistsException | FileSystemAlreadyExistsException e)
         {
             LOG.warn("ignore FileAlreadyExistsException: archiveURI {}, outputDir {}", archiveURI, outputDir);
```

Comment:
This message should have probably changed too.

Reply:
ah yes good catch.
done here https://github.com/jetty/jetty.project/pull/12415